src/views/tutorial-view/tutorial-view.module.css

```diff
@@ -0,0 +1,3 @@
+.featuredInCollections {
+  margin-top: 48px;
```

Comment:
Thoughts on adding the margin bottom as well? Or do you think that would be handled elsewhere? Perhaps its not needed, but its difficult to tell without the scaled layout. 

Seems like the designs have it as 48 - desktop, 50 - tablet, 32 - mobile. It's up to you. We can always add it later 👌

Reply:
Is the "edit on github" `StandaloneLink` what will be placed after this section? I poked through the designs for a minute but couldn't find a frame that specifies what'll come after these.

Reply:
☝️ both great callouts - i didn't noticed spacing was responsive, looks like there's other stuff I missed too 😅 I missed that in `NextPrevious` as well I think 😞 Will open a follow-up PR for that 👍

`Edit on GitHub` does appear to come after - [Figma link](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041) (should hopefully link directly to the artboard with the stuff on it; if not it's the `Learn components` page in the `Dev Portal` file).

Reply:
re: `margin-bottom`: It might be a weird MO, but my plan is to generally stick to `margin-top` for elements. I find it less confusing, I dunno 🤷 And then if an element is omitted from the flow, it "takes its spacing with it" (doesn't leave an accidental margin below itself). Let me know if it's cool if I take that approach, happy to adjust if we have more prescriptive guidance.

Reply:
I don't think that's weird at all! "takes its spacing with it" is a super sound mentality. I'm a fan ✨